core/src/perf_libs.rs

```diff
@@ -86,7 +89,21 @@ fn init(name: &str) {
 }
 
 pub fn init_cuda() {
-    init("libcuda-crypt.so")
+    let exe_dir = std::env::current_exe().expect("Unable to get executable path");
+
+    let libcuda_path: PathBuf = ["perf-libs", "libcuda-crypt.so"].iter().collect();
+
+    // Look for {.,..,...}/perf-libs/libcuda-crypt.so
```

Comment:
This starts to seem a bit risky to me. Need to be careful an attacker couldn't load a nefarious version of the library outside of a path you control.

Reply:
Good point.  With my latest commit, we now only look for `./perf-libs/libcuda-crypt.so` (which enabled me to remove `source target/perf-libs/env.sh` entirely as a freebie)